### PR TITLE
Update for new font render

### LINKED ISSUE #17: [Bug] Some Languages Showing empty squares in app
When I enter the language selection Menu in the app it seems that some of the languages are bugged and show empty boxes instead of the language symbols. When switching to those languages the missing symbols changes though.

For Example on the English setting the Korean Characters are Missing
2022012305001600-FABBE04B5BB8324A427618E6D5E2ED75

On the Korean Setting Some Chinese characters are missing
2022012305024200-FABBE04B5BB8324A427618E6D5E2ED75

and on the Chinese setting Korean is missing as well as the special characters from french and spanish. It also Breaks other things such as the spacing of Russian Characters.
2022012305030600-FABBE04B5BB8324A427618E6D5E2ED75

These issues aren't limited to just the language selection Menu unfortunately. If you have NSP/XCI/NSZ/XCZ files or folders in any of these languages it will mess up their text as well. Here's an example of that:
2022012305293900-FABBE04B5BB8324A427618E6D5E2ED75

### PR DESCRIPTION
**Pull request by R-YaTian**
Close #17 
Also: Update translations for Chinese-Simplified
Also: Fix warning when compilation
I also opened [a new PR for Plutonium](https://github.com/dezem/Plutonium/pull/1)
Please update the submodule manually. (:p
Test build (with latest libnx and libusbhsfs v0.2.6): [AtmoXL-Titel-Installer.zip](https://github.com/R-YaTian/AtmoXL-Titel-Installer/releases/download/untagged-c405ceb660dc25d25f97/AtmoXL-Titel-Installer-master.zip) It works all fine for me.

### CODE REVIEW

**Comment by dezem** [comment]
I merged it to a new branch and make further tests.
Thank you for your work! 😉

**Comment by dezem** [comment]
@R-YaTian many thanks for your contribution.
Works great with the updated Plutonium 👍

**Comment by R-YaTian** [comment]
> @R-YaTian many thanks for your contribution. Works great with the updated Plutonium 👍

You are welcome!

File: romfs/lang/zh-CN.json [hunk 1]
```diff
@@ -113,23 +113,23 @@
...
-            "preparing": "初始化安装过程...",
+            "preparing": "正在准备安装...",
             "failed": "安装失败 ",
             "failed_desc": "未完整安装的部分可以从系统设置中删除。",
             "complete": "安装完成",
             "desc0": " 个文件安装完成！",
             "desc1": " 安装完成！",
-            "downloading": "传输中……",
+            "downloading": "传输中...",
             "at": " 大约 "
```
Comment (WanderMax) [inline romfs/lang/zh-CN.json:123]: at 23.33 mb/s，I think "大约" is not that good.

Comment (R-YaTian) [reply romfs/lang/zh-CN.json:123]: > at 23.33 mb/s，I think "大约" is not that good.

Changes made by "e18cca3". How about "以"? Maybe I will fix it next time.

Comment (WanderMax) [reply romfs/lang/zh-CN.json:123]: good!

**Pull request merged**